_templates/package-generator/with-prompt/src/components/App/index.ejs.t

```diff
@@ -36,44 +35,59 @@ const paragraphStyle = css`
 `;
 
 const App = () => (
-  <>
+  <Fragment>
     <Global styles={BrandTheme} />
     <PageLayout
       teamTitle="<%=title%>"
       heroTitle="🚧 Site Under Construction 🚜"
       heroSubtitle="Playground for building out the <%=year%> <%=title%> team frontend"
       overlay
     >
-      <section className={cx(sectionBodyHeading, sectionMaxWidthSmall)}>
+      <section css={css`
```

Comment:
There's actually a nicer way to combine these styles:
`css={[sectionBodyHeading, sectionMaxWidthSmall]}`

[(described here, bottom code block)](https://emotion.sh/docs/composition)

Reply:
Crikey. I just didn't scroll far enough down. Fixing.